scss/tokens/_raw.scss

```diff
@@ -65,6 +65,65 @@ $ouds-elevation-y-500:  12px !default;
 // $ouds-elevation-y-600:  20px !default;
 // scss-docs-end ouds-raw-elevation
 
+// Grid
+
+// scss-docs-start ouds-raw-grid
+$ouds-grid-width-100: 360px !default; // 2xs
+$ouds-grid-width-200: 390px !default; // xs
+$ouds-grid-width-300: 480px !default; // sm
+$ouds-grid-width-400: 768px !default; // md
+$ouds-grid-width-500: 1024px !default; // lg
+$ouds-grid-width-600: 1440px !default; // xl
+$ouds-grid-width-700: 1680px !default; // xxl
+$ouds-grid-width-800: 1920px !default; // 3xl
+
+//$ouds-grid-min-width-100: 1 !default;
+//$ouds-grid-min-width-200: 390 !default;
+//$ouds-grid-min-width-300: 480 !default;
+//$ouds-grid-min-width-400: 736 !default;
+//$ouds-grid-min-width-500: 1024 !default;
+//$ouds-grid-min-width-600: 1320 !default;
+//$ouds-grid-min-width-700: 1640 !default;
+//$ouds-grid-min-width-800: 1880 !default;
+//
+//$ouds-grid-max-width-100: 389 !default;
+//$ouds-grid-max-width-200: 479 !default;
+//$ouds-grid-max-width-300: 735 !default;
+//$ouds-grid-max-width-400: 1023 !default;
+//$ouds-grid-max-width-500: 1319 !default;
+//$ouds-grid-max-width-600: 1639 !default;
+//$ouds-grid-max-width-650: 1680 !default;
+//$ouds-grid-max-width-700: 1879 !default;
+//$ouds-grid-max-width-800: 1920 !default;
+
+$ouds-grid-margin-100: 16 !default;
```

Comment:
Wondering if we shouldn't use a multiplicator based on `$dimension-base`. They've done it like this in OUDS Android for instance: https://github.com/Orange-OpenSource/ouds-android/blob/db96b330d4df815d802e773f8bfc707894c3c8d0/global-raw-tokens/src/main/java/com/orange/ouds/tokens/global/raw/GridRawTokens.kt#L24

Reply:
Should be done at the design level right ?

Reply:
That's now the case, I'll update the PR to reflect that.